webhook-bridge/svix-webhook-bridge.example4.yaml

```diff
@@ -0,0 +1,65 @@
+log_level: "debug"
```

Comment:
I think we probably want something in the middle. So probably the other as base, but taking in the input/output things from this one.

Reply:
Mashed the two up in `.example5.senders.yaml` and `.exampler5.receivers.yaml`. 

5th iteration feels pretty viable to me :rofl: